src/bson.rs

```diff
@@ -397,19 +397,19 @@ impl Bson {
         match *self {
             Bson::RegExp(ref pat, ref opt) => {
                 doc! {
-                    "$regex" => (pat.clone()),
-                    "$options" => (opt.clone())
+                    "$regex" => pat.clone(),
```

Comment:
Out of curiosity, why remove the parens/add trailing commas but not switch the "=>" to ":"? Is this a rustfmt thing?

Reply:
nope, just made these changes before I re-added colon support. i'll change these.